README.md

```diff
@@ -62,10 +71,14 @@ full list of requirements can be found by opening that script.
    [leaflet-usermarker] into the third party libraries and
    [resources](https://github.com/tourtrak/resources) into the resources
    directory.
-5. If you are not planning on building both Android and iOS apps, you should
+5. Make a copy of "config.json.template" named "config.json" and fill out the app configuration(config.json):
```

Comment:
It looks like we refer to files in several different ways (formatting wise - quotes, code, or inline).  Should we consider standardizing?

Probably outside this pull, but something to think about